### DISCUSSION TITLE
involvement of Christian

### DISCUSSION THREAD

**Discussion by luxus** [LunarVim/LunarVim#2847] [General]
Hi,
I think many people notice by now that @ChristianChiarulli doesn't contribute much to lunarvim since November https://github.com/LunarVim/LunarVim/commits?author=ChristianChiarulli
I wonder why it still sounds like on the landing page that he is the main dev. i know he started everything, but he moved on... but I don't like that all the contributor and especially  @kylo252 doesn't get mentioned and there are still Christian sponsor links in place.

Am I the only one that feels like that?

**Comment by ChristianChiarulli** [comment]
I do plan to be more involved in the future, I think adding sponsor links for @kylo252 is a good idea. These kind of discussions should probably take place in the lunarvim devs discord channel since it's easier for me to notice being pinged there. We can also get more visibility for regular contributors as well, what do you propose?
